config/google.js

```diff
@@ -1,7 +1,7 @@
 const googleCredentials = {
   clientId: process.env.GOOGLE_CLIENT_ID,
   clientSecret: process.env.GOOGLE_CLIENT_SECRET,
-  redirectUri: process.env.GOOGLE_REDIRECT_URI,
+  redirectUri: process.env.GOOGLE_REDIRECT_URL,
```

Comment:
Can we revert this change? The variable in class is redirectUri.

Reply:
> Can we revert this change? The variable in class is redirectUri.

OK, I thought it is a typo. `url -> uri` Will change it soon.